### PR TITLE
Add filesystem and pool name parameters to acquire_filesystem_symlinks

### PR DESCRIPTION
**Pull request by bgurney-rh**
Supersedes: #6 
Closes: #5

### CODE REVIEW

File: stratisd_cert.py
Comment (mulkieran) [inline stratisd_cert.py:95]: Whoops! I missed a spot. You should be able to put the ```disable``` after the open paren, like here https://github.com/stratis-storage/stratisd/blob/develop-2.3.0/tests/client-dbus/tests/udev/test_udev.py#L265.

**Review by mulkieran — CHANGES_REQUESTED** [review]
I missed a spot. Feel free to force push on this change.

File: stratisd_cert.py
Comment (bgurney-rh) [reply stratisd_cert.py:95]: I tried that before, and it didn't work:
```
************* Module stratisd_cert
stratisd_cert.py:94:0: C0330: Wrong hanging indentation before block (add 4 spaces).
    pool_name, filesystem_name, pool_path, filesystem_path
    ^   | (bad-continuation)
```

**Comment by mulkieran** [comment]
Let's wait until we merge the new black, to see if that gets rid of the pylint/black disagreement.

**Comment by bgurney-rh** [comment]
Rebased.

File: stratisd_cert.py
Comment (bgurney-rh) [reply stratisd_cert.py:97]: Now it seems to have worked; I updated it via a force-push.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**
